test/extended/testdata/bindata.go

```diff
@@ -21110,7 +21110,7 @@ var _testExtendedTestdataBuildsTestS2iBuildQuotaJson = []byte(`{
   "spec": {
     "resources": {
       "limits": {
-        "cpu": "60m",
+        "cpu": "400m",
```

Comment:
per team discussion in scrum .... this new value is still significantly less than what we would proscribe to a customer ... the quote @adambkaplan, proscribing an entire core for builds could be considered "best practice"

and certainly our iterating over this has proven the value is getting properly applied to the build object, which is the real intent of the s2i quota test :-)

Reply:
60m still works, too, but takes a lot of time to complete.

Reply:
yes fair clarification 

the current test timeout of 10min was bumped to 15min to allow for that, but we are going with this approach instead